UI/managelists.html

```diff
@@ -53,11 +53,32 @@
             </li>
        </ul>
     <div id="add_bckt_list_div">
-    <a id="btn_add_bckt" class="btn-floating btn-large waves-effect waves-light blue-grey lighten-1 center"><i class="material-icons">add</i></a>
```

Comment:
Lines are too long. Consider using a standard line width and indentation to make code more readable